packages/editor/src/components/post-title/index.js

```diff
@@ -186,6 +173,13 @@ const applyWithDispatch = withDispatch( ( dispatch ) => {
 		onUndo: undo,
 		onRedo: redo,
 		clearSelectedBlock,
```

Comment:
Do we still need to include `clearSelectedBlock` as a prop here on 175 now that it is only being used by `onSelect` on line 178?

Reply:
Indeed it's not needed anymore 👍